sjb/config/test_cases/test_branch_origin_web_console.yml

```diff
@@ -2,14 +2,40 @@
 parent: 'common/test_cases/origin_built_release.yml'
 extensions:
   actions:
+    - type: "script"
+      title: "determine binary and image version"
+      repository: "origin"
+      script: |-
+        if [[ "${PULL_BASE_REF}" = "enterprise-"* ]]; then
+          if [ "${PULL_BASE_REF}" == "enterprise-3.6" ]; then
+            echo "IMAGE_VERSION=v3.6.1" > IMAGE_VERSION_VAR
+          elif [ "${PULL_BASE_REF}" == "enterprise-3.8" ]; then
+            echo "IMAGE_VERSION=v3.9" > IMAGE_VERSION_VAR
+          else
+            echo "IMAGE_VERSION=${PULL_BASE_REF/enterprise-/v}" > IMAGE_VERSION_VAR
+          fi
+        else
+          echo "IMAGE_VERSION=latest" > IMAGE_VERSION_VAR
+        fi
+        source IMAGE_VERSION_VAR
+        sudo docker pull openshift/origin:"${IMAGE_VERSION}"
+        sudo docker create --name temp-container openshift/origin:"${IMAGE_VERSION}"
+        sudo docker cp temp-container:/usr/bin/oc ./
+        sudo docker rm temp-container
+        sudo mv -f ./oc /bin/oc
     - type: "script"
       title: "start openshift server"
       repository: "origin"
       script: |-
-        sudo yum --disablerepo=\* --enablerepo=origin-local-release,oso-rhui-rhel-server-releases install -y origin-clients
         wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
         sudo yum install ./google-chrome-stable_current_*.rpm -y
-        oc cluster up --version=latest --public-hostname=localhost --loglevel=5 --server-loglevel=5
+        source IMAGE_VERSION_VAR
+        oc version
+        if [ "${IMAGE_VERSION}" == "v3.9" ]; then
+          oc cluster up --version=v3.9 --public-hostname=localhost --loglevel=5 --server-loglevel=5
```

Comment:
Any reason not to use `--version="${IMAGE_VERSION}"` always and skip this if/else?

Reply:
`oc` will download the images based on its version. The `if` exception is because of an issue in our 3.9 images that prevents from starting the cluster.
I think that there was one other issue with one specific version, but cant recall what it was. Anyway will try it on my testing job for all the release versions and will update you

Reply:
Should explicitly write down why we are doing this in the job or here in the comments.

Reply:
We should be remove to remove this special case when we have a `v3.9` git tag since 3.9 cluster up will work without `--version`.